javalanglib/src/main/scala/java/lang/Class.scala

```diff
@@ -19,8 +19,9 @@ private trait ScalaJSClassData[A] extends js.Object {
   def newArrayOfThisClass(dimensions: js.Array[Int]): AnyRef = js.native
 }
 
-final class Class[A] private (private val data: ScalaJSClassData[A])
-    extends Object {
+final class Class[A] private (data0: Object) extends Object {
```

Comment:
Why does this take `Object` and not `js.Any`?

Reply:
Because `js.Any` would result in the signature `init___sjs_js_Any`, which is tight to Scala.js as a source language.